File: ProtoScaner.Server/Models/HistorialCambio.cs
using System;
using System.Collections.Generic;

namespace ProtoScaner.Server.Models;

public partial class HistorialCambio
{
    public int IdHistorial { get; set; }

    public int? IdUsuario { get; set; }

    public string? TablaModificada { get; set; }

    public int? IdRegistroModificado { get; set; }

    public string? Operacion { get; set; }

    public string? ValorAnterior { get; set; }

    public string? ValorNuevo { get; set; }

    public byte[] FechaMidificacion { get; set; } = null!;

    public virtual Usuario? IdUsuarioNavigation { get; set; }
}

File: protoscaner.client/src/types/HistorialCambio.ts
// src/types/HistorialCambio.ts
export interface HistorialCambio {
    idHistorial: number;
    idUsuario?: number;
    tablaModificada?: string;
    idRegistroModificado?: number;
    operacion?: string;
    valorAnterior?: string;
    valorNuevo?: string;
    fechaModificacion: Uint8Array; // Utilizamos Uint8Array para representar byte[]
}
